src/storage/src/test_util/descriptor_util.rs

```diff
@@ -94,6 +94,7 @@ impl RegionDescBuilder {
             row_key: self.key_builder.build().unwrap(),
             default_cf: self.default_cf_builder.build().unwrap(),
             extra_cfs: Vec::new(),
+            compaction_time_window: None, // TODO should we it to "RegionDescBuilder"
```

Comment:
Looks like `RegionDescBuilder` is only used in unit tests. So it depends on you. You can add `compaction_time_window` when you have to write the test.